functions/menu.bash

```diff
@@ -216,6 +218,8 @@ show_main_menu() {
       42\ *) migrate_installation "openHAB3" && openhabian_update "openHAB3";;
       *Downgrade\ to\ openHAB\ 2) migrate_installation "openHAB2" && openhabian_update "stable";;
       43\ *) openhab_shell_interfaces;;
+      50\ *) jsscripting_npm_install "openhab";;
+      51\ *) jsscripting_npm_install "openhab_rules_tools";;
```

Comment:
Same comment, if it's being installed by default we probably don't need this menu option.